docs/installation.rst

```diff
@@ -63,99 +63,16 @@ and after you are done with TARDIS you can deactivate::
 One does not need to recreate the environment, but simply activate it every time
 TARDIS is used.
 
```

Comment:
I would be more verbose here: Since TARDIS has reached a mature state, we recommend always installing the latest development version.